android/src/test/java/com/instabug/reactlibrary/RNInstabugAPMModuleTest.java

```diff
@@ -204,4 +206,29 @@ public void testSetFlowAttribute() {
         verify(APM.class, times(1));
         APM.endUITrace();
     }
+    @Test
+    public void givenCallback$isW3ExternalTraceIDEnabled_whenQuery_thenShouldCallNativeApiAndResolvePromise(){
```

Comment:
Greate effort, Here's a small request. Can we use the new naming convetions of unit test methods?